v6_carrier_py/wrapper.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pickle
+from io import StringIO
+
+import pandas as pd
+from SPARQLWrapper import SPARQLWrapper, CSV
+from vantage6.client import serialization
+from vantage6.tools import docker_wrapper
+from vantage6.tools.data_format import DataFormat
+from vantage6.tools.dispatch_rpc import dispact_rpc
+from vantage6.tools.util import info
+
+SPARQL_RETURN_FORMAT = CSV
+
+
+def sparql_wrapper(module: str):
+    info(f"wrapper for {module}")
+
+    # read input from the mounted inputfile.
+    input_file = os.environ["INPUT_FILE"]
+    info(f"Reading input file {input_file}")
+
+    # TODO: _load_data handles input deserialization. It should be a public function
+    input_data = docker_wrapper._load_data(input_file)
+
+    query = input_data['query']
+
+    # all containers receive a token, however this is usually only
+    # used by the master method. But can be used by regular containers also
+    # for example to find out the node_id.
+    token_file = os.environ["TOKEN_FILE"]
+    info(f"Reading token file '{token_file}'")
+    with open(token_file) as fp:
+        token = fp.read().strip()
+
+    endpoint = os.environ["DATABASE_URI"]
+
+    # Workaround because the endpoint is automatically prefixed with the data folder. However this does not make
+    # sense for a sparql endpoint.
+    endpoint = _fix_endpoint(endpoint)
+
+    info(f"Using '{endpoint}' as triplestore endpoint")
+
+    data = query_triplestore(endpoint, query)
+
+    # make the actual call to the method/function
+    info("Dispatching ...")
+    output = dispact_rpc(data, input_data, module, token)
+
+    # write output from the method to mounted output file. Which will be
+    # transfered back to the server by the node-instance.
+    output_file = os.environ["OUTPUT_FILE"]
```

Comment:
Maybe you can factor this out into a function: write_output?

Reply:
This code is actually a copy of code I wrote for the regular vantage6 wrapper. Ideally I should make that into a function, along with some other reusable stuff. I'd say that belongs in a different PR because I first have to create a PR for fixing this in the regular vantage6 wrapper

Reply:
Agree!